README.md

```diff
@@ -1 +1,53 @@
-# 500apps-public-gtm-pbxplus
+# PBX Plus by 500apps
+PBX Plus is an advanced communications system and IVR for your business needs. Configure the call flow to allow for automatic forwarding so that employees can spend their time on building long-lasting customer relationships. Add new numbers, change call flows and use existing numbers so your customers are sure to reach you each and every time they call. Gain a better understanding of your customers and your call flow process with the ability to analyse the data and generate call reports. 
+
+Understand decisions made by customers so you can make educated business decisions based on what options the customer chooses on calls. Multi-level menus, unlimited call minutes, business hours, music-on-hold, dial by name, outbound dialer, conference rooms, forwarding, voicemail transcriptions, click to call, and much more at one place with PBX Plus.
+
+[Get Started](https://infinity.500apps.com/Pbxplus) with easy to use and configurable virtual PBX application. 
+
+Automate call processes, optimize team productivity, and improve customer experience with best-in-class virtual phone system.- PBX Plus(https:/pbxplus.com)
+Sign in to your Google account.
+
+Go to the Google Tag Manager home and click on the Create Account button on the right
+Enter the account as (Account name) and container name( Website URL).
+The container holds all your tags, such as the Google Analytics tag, Google Ads tag, or PBX Plus tag. Choose the target platform to web and click on the Continue button.
+
+Read through the GTM Terms of Service. Click Yes to accept them and continue.
+You will get a container with a code snippet. Copy the code and paste it into every page of your site in the recommended places. Click Ok to close the code snippet box and finish setting up the GTM manager and your first container.
+
+### Installation 
+
+1. Adding the PBX Plus tag to your website once you're logged in to your account with the container, it's time to add a new tag with the PBX Plus tracking code inside it. To do so, follow these steps
+In the Google Tag Manager Home, click on the New Tag tile.
+
+
+2. Click on the Tag Configuration tile and search for our community template 'PBX Plus by 500apps' in the 'Community Template Gallery'.
+
+
+3. Click on ‘Add to workspace’ to select the template and add it to your workspace
+After the template is added to your workspace, the tag is automatically added to your tag configuration.
+
+
+4. Find the generated PBX Plus key for the department , copy and place the key in the field provided in the Tag Configuration. You can find the PBX Plus key in the Share Script slide bar in the PBX Plus web application for each department. Just click on Google Tag Manager in the Share Script slide bar and you can find the key and the instructions.
```

Comment:
instructions are not step by step. Where is department

Try above steps and see it works. It will never